i18n/ru/docusaurus-plugin-content-docs/current/commands/browser/saveState.mdx

```diff
@@ -0,0 +1,47 @@
+import Admonition from "@theme/Admonition";
+
+# saveState
+
+## Обзор {#overview}
+
+Команда для сохранения состояния страницы (cookies, local и session storages).
+
+<Admonition type="warning">
+    Если вы используете `webdriver` automation protocol вы должны установить `webSocketUrl: true` в `desiredCapabilities` для браузера в конфиге.
+    Для `devtools` protocol не нужно дополнительных настроек.
+</Admonition>
+
+## Использование {#usage}
+
+Команда вернёт объект включающий cookie, localStorage и sessionStorage.
```

Comment:
I think it would be nice to have "Command Parameters" table like we do on other pages, e.g. https://testplane.io/docs/v8/commands/browser/setMeta/

Reply:
Fixed